code/renderers/preact/src/config.ts

```diff
@@ -1,3 +1,5 @@
+import { parameters as docsParams } from './docs/config';
+
 export { renderToDOM, render } from './render';
 
-export const parameters = { framework: 'preact' as const };
+export const parameters = { framework: 'preact' as const, ...docsParams };
```

Comment:
Would you mind separating this into another PR? This way you can tag it as `app:preact` `feature-request` and document about it, given that it's a breaking change. Then a release can be linked to that PR, rather than to a PR that is doing maintenance work

Reply:
Good idea, I will do.

Reply:
Thank you!